### PR TITLE
Update cw-plus to 0.11

### PR DESCRIPTION
**Pull request by orkunkl**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -42,11 +42,11 @@ optimize = """docker run --rm -v "$(pwd)":/code \
...
 cosmwasm-storage = { version = "1.0.0-beta" }
-cw-storage-plus = "0.10"
-cw2 = "0.10"
-schemars = "0.8.3"
-serde = { version = "1.0.127", default-features = false, features = ["derive"] }
-thiserror = { version = "1.0.26" }
+cw-storage-plus = "0.11"
+cw2 = "0.11"
+schemars = "0.8"
+serde = { version = "1.0.12", default-features = false, features = ["derive"] }
```
Comment (ethanfrey) [inline Cargo.toml:48]: Why lowering the version of serde and thiserror?

I think we bumped them earlier to ensure we got the advantages.
Is there anyone that requires such old versions?

File: src/contract.rs [hunk 2]
```diff
@@ -100,7 +100,7 @@ mod tests {
 
     #[test]
     fn increment() {
-        let mut deps = mock_dependencies(&coins(2, "token"));
+        let mut deps = mock_dependencies();
```
Comment (ethanfrey) [inline src/contract.rs:103]: Here don't just remove them. There is `mock_dependencies_with_balance(&coins(2, "token"))` that should have the same effect.

**Review by ethanfrey — CHANGES_REQUESTED** [review]
Thanks for taking this on.
Two questions.

File: Cargo.toml (hunk 1 above)
Comment (orkunkl) [reply Cargo.toml:48]: Same as https://github.com/CosmWasm/cw-tokens/pull/5#discussion_r776233594.
```
T/cw-template.F7vJMjN1w/testgen-local  update-0.11 ✗                                                                                                ◒  
▶ cargo tree | ag thiserror
│   │   │   └── thiserror v1.0.26
│   │   │       └── thiserror-impl v1.0.26 (proc-macro)
│   │   └── thiserror v1.0.26 (*)
│   ├── thiserror v1.0.26 (*)
└── thiserror v1.0.26 (*)
```

Comment (ethanfrey) [reply Cargo.toml:48]: Yes, but this means it accepts any serde from 1.0.12 on, not 1.0.100 on.

I remember some issues with very old serde (like 2 years ago), so I would rather provide a bit newer minimum version, like 1.0.83 at least

**Review by ethanfrey — APPROVED** [review]
Looks good.

I would still stick with the higher versions of serde and thiserror.

**Review by JakeHartnell — APPROVED** [review]

**Pull request merged**